include/nlohmann/detail/conversions/from_json.hpp

```diff
@@ -30,8 +30,7 @@
 #include <nlohmann/detail/value_t.hpp>
 
 NLOHMANN_JSON_NAMESPACE_BEGIN
-namespace detail
-{
+namespace detail {
```

Comment:
`.clang-format` has BraceWrapping/AfterNamespace set to false, which is the opposite of the general format of this repo.

Reply:
Thanks - I'll adjust.